src/modules/token/fungible/hooks/queries/useTokenAccountExists.test.js

```diff
@@ -0,0 +1,29 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { mockTokensAccountExists } from '@token/fungible/__fixtures__';
+import { queryWrapper as wrapper } from 'src/utils/test/queryWrapper';
+import { useTokenAccountExists } from './useTokenAccountExists';
```

Comment:
look like we don't have usage for it  and it has overlab with useCheckInitializedAccount

Reply:
@soroushm I still kept it for future use. I can remove it if you feel it should be removed